### ISSUE TITLE
Many to many association count

### ISSUE THREAD

**Issue by mabashian** [randallmeeker/SailsBluePrintActions#1]
I've implemented the solution (overwriting find.js in api/blueprints) outlined here in this repository and it works great for getting the total on a particular model ex:

`GET - /user`

What I would really like to do is get the count of a child association.

I have a model (user) that has an association with another model (phone). This association is many to many. I want to use the following call to get a subset of the phone objects associated with a particular user:

`GET - /user/:userId/phones`

Overwriting find.js doesn't seem to impact the above call.  Is there an elegant solution for getting the total number of phones associated with a user or do I need to write a custom route each time I need to get this information?

**Comment by mikedevita** [comment]
I too would find this useful.. given a query to `/checklist` with your blueprint you'd expect something like this..

``` javascript
[{
  info: {
    start: 0,
    end: 30,
    limit: 30,
    total: 3,
    criteria: {}
  },
  items: {
    name: 'checklist1',
    statuses: [
      { ... },
      { ... },
      { ... },
      { ... },
    ]
  }
}];
```

that works great.. what I'd like to see is the `info` object pushed to all other child associations like so..

This would allow me to then only make one call to `/checklist` to get information like, how many checklists in total, and also how many items in total each checklist has.

``` javascript
{
  "meta": {
    "start": 0,
    "end": 30,
    "limit": 30,
    "total": 1,
    "criteria": {}
  },

  //model results are inside of "items"
  "items": [
    {
      "name": "checklist1",

      //statuses is an association
      statuses: {
        "meta": {
          "start": 0,
          "end": 30,
          "limit": 30,
          "total": 3,
          "criteria": {}
        },
        "items": [
          { ... },
          { ... },
          { ... },
        ]
      }
    }
  ]
}
```

**Comment by randallmeeker** [comment]
These are really great ideas. If anyone wants to work on it, I'll take PR. I'm off of sails dev till after labor day.

**Comment by mikedevita** [comment]
Yeah I took a peak. Unfortunately at first glance it looks like you'd need to modify actionUtil.
